.husky/post-checkout

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+if ! command -v git-lfs >/dev/null 2>&1; then
+    cat >&2 << EOF
+This repository is configured for Git LFS but 'git-lfs' was not found on your path.
+Please install git-lfs through: https://github.com/git-lfs/git-lfs#installing
```

Comment:
I tweaked this message slightly from the message git-lfs adds by default to link to the installation page.